tests/test_trainer.py

```diff
@@ -33,16 +33,21 @@ def output_logs_dir():
         yield temp_dir
 
 
-def test_mobilenet_v1_on_catdog_datasets(catdog_dictionary, catdog_train_dataset_dir, catdog_val_dataset_dir, output_model_dir, output_logs_dir):
+@pytest.fixture('session')
+def catdog_num_classes():
```

Comment:
You can pass `catdog_dictionary` into this method and then `return len(catdog_dictionary)`